src/fetch_issues.py

```diff
@@ -0,0 +1,99 @@
+import csv
+import os
+import requests
+
+def get_college_from_label(issue):
+    college_list = [
+        'AMRT',
+        'COEP',
+        'DEI',
+        'DLBG',
+        'IIITH',
+        'IITB',
+        'IITD',
+        'IITG',
+        'IITK',
+        'IITKGP',
+        'IITR',
+        'NITK'
+    ]
+    
+    ##in label check which college is present, labels are like "Inappropriate,IIITH...etc"
+    for label in issue.get("labels", []):
+        for college in college_list:
+            if college in label["name"]:
+                return college
+        
+    return None
+    
+
+def fetch_issues():
+    # GitHub repository details
+    repo_owner = "virtual-labs"  # Replace with the repo owner's username/org
+    repo_name = "bugs-virtual-labs"     # Replace with the repository name
```

Comment:
In the new PR, this will be read in only one place ie in src/run_filter.py since src/fetch_issues.py will be removed. Again, backup will be done in AppScript so the script is unnecessary now. Should I still switch to config file?